tests/migrations/alembic.ini

```diff
@@ -0,0 +1,51 @@
+[alembic]
```

Comment:
shouldn't we use the actual `alembic.ini`? because if someone will change the original one, the tests won't "test" the real case.